code/modules/projectiles/projectile.dm

```diff
@@ -127,6 +127,10 @@
 	var/reflect_range_decrease = 5			//amount of original range that falls off when reflecting, so it doesn't go forever
 	var/is_reflectable = FALSE // Can it be reflected or not?
 
+	/*###DR ADDITIONS###*/
+	var/power_armor_blocks = FALSE //Does power armor stop this bullet?
```

Comment:
This is just a bad idea. You should just use the `armour_penetration` var available on all subtypes of `/obj/item`. It ranges from 0 to 1; find a sufficient threshold to block bullets under. You could even set the threshold on each power armour subtype to give it varying amounts of armour toughness; some block anything under 80%, while some only block anything under 20%.

As it stands, adding an entirely new value to bullets is just silly.